src/lib/types/Enums.ts

```diff
@@ -11,3 +11,10 @@ export const enum PluginHook {
 	PreLogin = 'preLogin',
 	PostLogin = 'postLogin'
 }
+
+export const enum BucketType {
+	Channel,
+	Global,
+	Guild,
+	User
+}
```

Comment:
Please document the enum and the members, so users can know what this is for.

Reply:
Resolved in d23b0de